themes/V3/5ePHB/style.less

```diff
@@ -831,6 +831,9 @@ h6,
 			text-decoration : none;
 			&:hover { text-decoration : underline; }
 		}
+		h1, h2, h3, h4, h5, h6 {
+			--TOC: exclude !important;
```

Comment:
Uncomfortable with this `!important`.

The higher specificity of `.toc h1` (etc) should be sufficient for the `exclude` to apply.

Reply:
Can certainly drop it if its unneeded

Reply:
Dropped